forem/forem
# Update Rubocop Version to 1.42.0

## What type of PR is this? (check all applicable)


## Description
Updating the Rubocop gem to the latest to workaround some erb_lint errors that we were receiving. For more context read through https://forem-team.slack.com/archives/C7GE84DEJ/p1673272052854839

## Related Tickets & Documents



- Related Issue #
- Closes #

## QA Instructions, Screenshots, Recordings

Run `rubocop -d /Users/ridhwana/Projects/Forem/forem/app/views/admin/comments/index.html.erb` on main and see the errors that pop up. 

```
app/views/admin/comments/index.html.erb:1:1: E: Lint/Syntax: unexpected token tLT
(Using Ruby 3.0 parser; configure using TargetRubyVersion parameter, under AllCops)
<%= paginate @comments %>
^

1 file inspected, 1 offense detected
```

Now run it on this branch and there will be no rubocop errors.

### UI accessibility concerns?

_If your PR includes UI changes, please replace this line with details on how
accessibility is impacted and tested. For more info, check out the
[Forem Accessibility Docs](https://developers.forem.com/frontend/accessibility)._

## Added/updated tests?

- [x] No, and this is why: library update

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Storybook](https://storybook.forem.com/) (for Crayons components)
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
      or in a [forem.dev](http://forem.dev) post
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] Are there any post deployment tasks we need to perform?

## [optional] What gif best describes this PR or how it makes you feel?

![alt_text](https://media.giphy.com/media/1imXktPlcuj9N8rBKN/giphy-downsized-large.gif)
